migrations/versions/bb3d7586a096_createsingletable.py

```diff
@@ -0,0 +1,47 @@
+"""CreateSingleTable
+
+Revision ID: bb3d7586a096
+Revises: a72f117515c5
+Create Date: 2023-10-24 14:46:03.868952
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+
+
+# revision identifiers, used by Alembic.
+revision = "bb3d7586a096"  # pragma: allowlist secret
+down_revision = "a72f117515c5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
```

Comment:
I think we need another PR after this gets merged to actually clean up the old tables. We can't forget or someone in the future will be super confused.

Reply:
Yeah definitely. This migration can be an in-between migration where both options are available.

Reply:
Can we add a TODO in this file?